api/v1alpha1/selfnoderemediation_webhook.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+)
+
+var (
+	webhookRemediationLog = logf.Log.WithName("selfnoderemediation-resource")
+)
+
+func (r *SelfNodeRemediation) SetupWebhookWithManager(mgr ctrl.Manager) error {
+
+	if err := initOutOfServiceTaintSupportedFlag(mgr.GetConfig()); err != nil {
```

Comment:
This call is duplicate now which at least isn't nice (and just relying here that it's called at the other webhook also isn't great) . Would it make sense to move this to main and pass the flag as arg? Or into its own file here and  just call init from main? 🤔

Reply:
Indeed, good catch.
I didn't want to create a new file in an already crowded package, and I was worried about cycle dependency setting flags so I ended up putting it in `taints` file.

Reply:
>  I ended up putting it in taints file

👍🏼 

/lgtm